pkg/authorization/audit_logger.go

```diff
@@ -52,6 +53,10 @@ func NewAuditLogger(auditPrefix string, delegate authorizer.Authorizer) authoriz
 			auditPrefix+"/"+auditReason, auditReasonMsg,
 		)
 
+		// Note: this deviates from upstream which doesn't log audit reasons.
+		// We should rethink if this should stay.
+		klog.V(5).Info(auditReasonMsg)
```

Comment:
our tests run V(4). Would suggest to use the same here.

Reply:
changed to V(4)

Reply:
i did, V(4) is in https://github.com/kcp-dev/kcp/pull/2442/files#diff-bce2789a10daa6a0f8a8fadb01ddbffa4a2db46d9cc62061ba01632822b941c6R75